learn/getting_started/getting_ready_for_production.md

```diff
@@ -0,0 +1,67 @@
+# Chapter 3: Getting ready for production
+
+This chapter will help you get ready to use Meilisearch in production by covering topics such as instance-wide options, data backup, and security.
+
+## Configuration options
+
+Meilisearch allows you to configure your entire instance through **[environment variables](/reference/features/configuration.md#configuring-an-instance-with-environment-variables)** before launch or **[command-line options](/reference/features/configuration.md#configuring-an-instance-with-command-line-options)** at launch.
+
+Meilisearch has configuration options for many critical actions, such as:
+
+- Changing the database path
+- Starting Meilisearch in a development or production environment
+- Setting a master key to protect API endpoints
+- Disabling telemetry (enabled by default)
+
+You can read about all of them in our [configuration guide](/reference/features/configuration.md).
+
+## Data backup
+
+Meilisearch offers two options for backing up your data: `dumps` and `snapshots`.
+
+Dumps export data in a raw unprocessed form and can be used to migrate your database between different tools or Meilisearch versions. A dump isn't an exact copy of your database—it is closer to a blueprint that allows you to rebuild an identical database.
```

Comment:
I did not know dumps could be used for anything besides Meilisearch. Can we list one or two examples of third-party tools?

Reply:
@dichotommy any suggestions?